homeassistant/components/tractive/__init__.py

```diff
@@ -36,10 +37,10 @@
     TRACKER_POSITION_UPDATED,
 )
 
-PLATFORMS = ["binary_sensor", "device_tracker", "sensor", "switch"]
+PLATFORMS: Final = ["binary_sensor", "device_tracker", "sensor", "switch"]
 
 
-_LOGGER = logging.getLogger(__name__)
+_LOGGER: Final = logging.getLogger(__name__)
```

Comment:
We don't need to mark all constants as final. It's only needed in some cases, eg to be able to use a constant string in a typed dict context, or similar.

Reply:
OK, will change this.

Reply:
https://github.com/home-assistant/core/pull/57000